config/config.go

```diff
@@ -43,7 +42,6 @@ type Config struct {
 	RedirectSecret               string        `envconfig:"REDIRECT_SECRET" json:"-"`
 	ReleaseCalendarControllerURL string        `envconfig:"RELEASE_CALENDAR_CONTROLLER_URL"`
 	ReleaseCalendarEnabled       bool          `envconfig:"RELEASE_CALENDAR_ENABLED"`
-	ReleaseCalendarRoutePrefix   string        `envconfig:"RELEASE_CALENDAR_ROUTE_PREFIX"`
 	UseNewReleaseCalendar        bool          `envconfig:"USE_NEW_RELEASE_CALENDAR"`
```

Comment:
These seem very similar? Why do we have two? Could we just have one?

Reply:
i.e. 

```ReleaseCalendarEnabled```
and 
```UseNewReleaseCalendar```

Reply:
Done